modules/administration/pages/recurring-actions.adoc

```diff
@@ -0,0 +1,76 @@
+
+* If you are the {productname}-admin you can access all recurring actions from menu:Admin [Organizations] then select the organization you want to manage and navigate to the menu:States [Recurring States] tab.
+
+=== List of all Recurring Actions
+
+If you want to see all recurring action schedules that you have access to navigate to menu:Schedule [Recurring Actions].
+
+[NOTE]
+====
+On deleting a user all schedules that the user created will be deleted as well.
+====
+
+== Creating a new Recurring Action
+
+Depending on the target you want to create an action for navigate to the before mentioned place.
+
+* Click the btn:[Create] button.
+* Enter a name for the schedule you want to create.
+* Chose the frequency you want the action to be executed in. You have the following options.
+** [guimenu]``Hourly:`` Select the minute of the hour that you want the action to be executed. This will execute the action every hour at the specified minute.
+** [guimenu]``Daily:`` Select the time at which the action should be executed every day.
+** [guimenu]``Weekly:`` Select the day of the week and the time of the day. The action will be executed every week at the specified time.
+** [guimenu]``Monthly:`` Select the day of the month and the time of the day. The action will execute every month at the chosen time.
+** [guimenu]``Custom Quartz format:`` If you can't select the frequency you desire with the above pickers you have the option to enter a custom quartz string. e.g. "0 15 2 ? * 7" (without the quotes and each number separated by space)
+* You can chose whether you want the action to be executed in [guimenu]``Test mode``.
+* Once you are confident about the data you entered press the btn:[Create Schedule] button.
+* You will be redirected back to the list of existing actions. Your newly created action should have appeared here as well.
+
+[NOTE]
+===
+Creating a new recurring action from the list of all actions is not possible, please create a new action from the respective location.
+===
+
+== Disabling a Recurring Action
+
+From the list of recurring actions you have the option to disable an action without deleting it. Just click the slider in the [guimenu]``Active`` column. You can re enable it at any time. The action will stop being executed as long as it is disabled.
+
+== Recurring Action Details
+
+You can view the details about your action from the action list. Click the first of the three buttons displayed in the last column of the action you want to view.
+
+== Edit Recurring Actions
+
+To edit an existing recurring action select the second of the three buttons in the last column of the action that you want to edit.
+
+This will redirect you to the page for creating/editing actions pre filled with the current properties of your action. Please refer to the [guimenu]``Creating a new Recurring Action`` section for guidance.
```

Comment:
```suggestion
This will redirect you to the page for creating and editing actions prefilled with the current properties of your action.
For more information, see the [guimenu]``Creating a new Recurring Action`` section.
```